### PR TITLE
source-revenuecat contribution from topefolorunso

### PR DESCRIPTION
**Pull request by topefolorunso**
## What

This PR adds a new connector for source RevenueCat (source-revenuecat).

Contributor-provided connector description:

RevenueCat is a powerful and reliable in-app purchase server that makes it easy to build, analyze, and grow your subscriber base whether you&#39;re just starting out or already have millions of customers.
## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `api_key` | `string` | API Key. API key or access token |  |
| `start_date` | `string` | Start date.  |  |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| projects | id | No pagination | ✅ |  ✅  |
| apps | id | No pagination | ✅ |  ✅  |
| metrics_overview | id | No pagination | ✅ |  ❌  |
| customers | id | No pagination | ✅ |  ✅  |
| entitlements | id | No pagination | ✅ |  ✅  |
| entitlements_products | id.entitlement_id | No pagination | ✅ |  ✅  |
| offerings | id | No pagination | ✅ |  ✅  |
| offerings_packages | id | No pagination | ✅ |  ✅  |
| offerings_packages_products | id.package_id | No pagination | ✅ |  ✅  |
| products | id | No pagination | ✅ |  ✅  |

</details>

## Reviewer checklist
- [x] Run `/format-fix` to resolve any formatting errors
- [x] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

**Comment by natikgadzhi** [comment]
@topefolorunso this is NICE! But, it needs purchases, invoices, and subscriptions streams. I know it's difficult to make test data for them, so it's ok to override and push the connector even though you don't have data to validate.

If it will refuse to push a PR, you can export the yaml file and add it to this PR manually

**Comment by topefolorunso** [comment]
Streams added @natikgadzhi. Please review again

**Review by natikgadzhi — APPROVED** [review]

File: airbyte-integrations/connectors/source-revenuecat/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
+  remoteRegistries:
+    pypi:
+      enabled: false
+      packageName: airbyte-source-revenuecat
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/source-declarative-manifest:4.6.2@sha256:f5fcd3d4703b7590b6166a7853c5ed1686731607cd30a159a8c24e2fe2c1ee98
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-revenuecat/metadata.yaml:16]:
```suggestion
    baseImage: docker.io/airbyte/source-declarative-manifest:5.7.3@sha256:4c4a567211fbcdf42805f261020bf5be46454bfeba781a92d30fe414ef964212
```

**Review by natikgadzhi — APPROVED** [review]

**Pull request merged**